### ISSUE TITLE
Translation triggers after Text element is closed

### ISSUE THREAD

**Issue by Channeira** [bbepis/XUnity.AutoTranslator#16]
In A-1 (started with --verbose) I'm seeing the following behaviour:
- Text is created (no translation is triggered)
- (time between is independent, tested [1s, 5s, 60s])
- Text is closed
- Translation is triggered immediatly afterwards

Reloading the same scene. Translation is now cached.
- Text is created
- Text is replaced by translation

I've tried a Manual hook (ALT+U) with the Dialog window ("BookManager") open without effect. So far I've implemented a few modifications to `TraverseChildrenManualHook` to understand the reason for this better but have no solution for now.

The translation is consistently triggered after the dialog box is closed.

Game: A-1 (Guilty Hell)
Tested AutoTranslator versions: 2.5.0, 2.15.4

Root hierarchy with dialog box open:
```
BookManager(Clone) [UI]                                                          RectTransform, Canvas, CanvasScaler, GraphicRaycaster, AiriBookManager
 TalkPage(Clone) [UI]                                                                         RectTransform, PageBehaviour, PageDisplay, FontSizeFitter
  Background [UI]                                                                                                  RectTransform, CanvasRenderer, Image
  Name [UI]                                                                                                         RectTransform, CanvasRenderer, Text
  Image [UI]                                                                                                       RectTransform, CanvasRenderer, Image
  TextLine1 [UI]                                                                                                    RectTransform, CanvasRenderer, Text
  TextLine2 [UI]                                                                                                    RectTransform, CanvasRenderer, Text
  ChoiceUI [UI]                                                                                                                           RectTransform
   Text [UI]                                                                                                              RectTransform, CanvasRenderer
    Text1 [UI]                                                                                                      RectTransform, CanvasRenderer, Text
    Text2 [UI]                                                                                                      RectTransform, CanvasRenderer, Text
    Cursor1 [UI]                                                                                                   RectTransform, CanvasRenderer, Image
    Cursor2 [UI]                                                                                                   RectTransform, CanvasRenderer, Image
  TextLineSizeBase [UI]                                                                                             RectTransform, CanvasRenderer, Text
[...]
```

**Comment by Channeira** [comment]
After a bit of testing I can confirm that only after the last dialog has been closed the last line of dialogue will be translated. It seems the OnTextChange triggers are not working despite the components getting updated with new text.

**Comment by gravydevsupreme** [comment]
Back when this issue was first created, I tried to determine the root cause of why it did not work, when it seemed like it should. I was unable to determine exactly why and for that I apologize. Now that the game has an official translation, I assume this issue is no longer relevant.

I am therefore closing the issue.
